### DISCUSSION TITLE
nuxt import error when upgrading to nuxt-vuefire 0.4.1

### DISCUSSION THREAD

**Discussion by MarkSonn** [vuejs/vuefire#1460] [Questions and Help]
i'm upgrading all my packages and get the following error when upgrading from `nuxt-vuefire 0.4.0` to `nuxt-vuefire 0.4.1`:
```[nuxt] `#imports` should be transformed with real imports. There seems to be something wrong with the imports plugin.```

does anyone know what it means?

**Comment by posva** [comment]
This was updated in https://github.com/vuejs/vuefire/pull/1457. I suppose you also need to update Nuxt

**Reply by MarkSonn** [reply]
that's not it in my case unfortunately, i made this post becase i upgraded every single package (including nuxt which sits at  `"nuxt": "^3.8.1"`) and the only package that caused an issue was `nuxt-vuefire 0.4.1`. would you like me to raise a github issue?

**Reply by posva** [reply]
On Nuxt, yes

**Reply by MarkSonn** [reply]
what exactly would the issue for nuxt be, sorry i don't really understand what is going wrong 😅 i feel like after mentioning the issue only happens when i upgrade nuxt-vuefire they would ask me to raise the issue here

**Reply by posva** [reply]
I tried to reproduce it and I don't have any error. I suppose you have an outdated cache somewhere, maybe `.nuxt`. If the problem persist, it's worth trying to boil it down following nuxt instructions but even with VueFire in it, it should be fine given https://github.com/nuxt/nuxt/pull/23951. But I really think this is a cache or config issue.
Cheers

**Reply by i4innovationnet** [reply]
I started a project from scratch using latest Nuxt and Vuefire versions and I still have the same warning. :\

**Reply by chilledtonic** [reply]
Would like to concur that I created a brand new project from scratch using nuxt, and the advice on [Vuefires documentation](https://vuefire.vuejs.org/nuxt/getting-started.html), and am having the exact same error. All packages are up to date.
